src/aks-preview/azext_aks_preview/tests/latest/test_decorator.py

```diff
@@ -2278,6 +2278,78 @@ def setUp(self):
         self.models = AKSPreviewModels(self.cmd, CUSTOM_MGMT_AKS_PREVIEW)
         self.client = MockClient()
 
+    def test_update_outbound_ips(self):
+        mc_1 = self.models.ManagedCluster(
+            location="test_location",
+            network_profile=self.models.ContainerServiceNetworkProfile(
+                load_balancer_profile=self.models.lb_models.get(
+                    'ManagedClusterLoadBalancerProfile'
+                )(
+                    outbound_ip_prefixes=self.models.lb_models.get(
+                        "ManagedClusterLoadBalancerProfileOutboundIPPrefixes"
+                    )(
+                        public_ip_prefixes=["id1", "id2"],
+                    )
+                )
+            )
+        )
+        self.client.get = Mock(return_value=mc_1)
+        dec_1 = AKSPreviewUpdateDecorator(
+            self.cmd,
+            self.client,
+            {
+                "name": "test_cluster",
+                "resource_group_name": "test_rg_name",
+                "load_balancer_outbound_ip_prefixes": "id3,id4",
+            },
+            CUSTOM_MGMT_AKS_PREVIEW,
+        )
+
+        mc = dec_1.fetch_mc()
+        mc = dec_1.update_load_balancer_profile(mc)
+        self.assertListEqual(
+            [pip.id for pip in mc.network_profile.load_balancer_profile.outbound_ip_prefixes.public_ip_prefixes], ["id3", "id4"])
+        self.assertIsNone(
+            mc.network_profile.load_balancer_profile.managed_outbound_i_ps)
+        self.assertIsNone(
+            mc.network_profile.load_balancer_profile.outbound_i_ps)
+
+    def test_update_outbound_ips(self):
```

Comment:
Aha, at first glance, I thought this is exactly the same test case as above. Please change the function name, otherwise the previous one won't be executed.